tests/test_completion.py

```diff
@@ -797,3 +799,106 @@ def test_additional_edits_if_session_does_not_have_the_resolve_capability(self)
             completion_items=[completion_item],
             insert_text='',
             expected_text='import ghjk;\nghjk')
+
+
+class ItemDefaultTests(CompletionsTestsBase):
```

Comment:
Inherit from `unittest.TestCase` since we don't need to set up LSP on a file for those tests.